docs/upgrading/v1.1-v1.2.md

```diff
@@ -0,0 +1,135 @@
+# v1.1 to v1.2
+
+Follow the [Regular Upgrading Process](./README.md).
+
+## Upgrading Notable Changes
+
+### Introduced `karmada-search` component
```

Comment:
We don't need to mention the `karmada-search` component here, as the component is optional.